lib/monobank/personal/corporate_webhook.rb

```diff
@@ -0,0 +1,15 @@
+require 'monobank/personal/webhook'
+
+module Monobank
+  module Personal
+    class CorporateWebhook < Webhook
```

Comment:
questionable inheritance chain 🤷‍♂️ , open to suggestions

Reply:
cannot provide better solution 🤷‍♂️